scripts/buildinputs/go.mod

```diff
@@ -0,0 +1,60 @@
+module dockerfile
+
+go 1.22.0
```

Comment:
This is okay, just checking why not 1.23 ? It is different to what we use in odh/kubeflow anyway. Or different reason?

Reply:
No particular reason, anything > 1.21 should be fine, I guess I had 1.22 around so that's what ended up here.